src/main/java/uk/gov/hmcts/reform/pip/model/publication/ListType.java

```diff
@@ -155,7 +155,15 @@ public enum ListType {
                                          "King’s Bench Masters Daily Cause List", false, true),
     SENIOR_COURTS_COSTS_OFFICE_DAILY_CAUSE_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES),
     MAYOR_AND_CITY_CIVIL_DAILY_CAUSE_LIST(NATIONAL, CFT_IDAM, ALL_VERIFIED_THIRD_PARTY_CFT_ROLES, null,
-                                          "Mayor & City Civil Daily Cause List", false, true);
+                                          "Mayor & City Civil Daily Cause List", false, true),
+    BIRMINGHAM_ADMINISTRATIVE_COURT_DAILY_CAUSE_LIST(NATIONAL, PI_AAD, ALL_VERIFIED_THIRD_PARTY_PRESS_ROLES,
+        null,"Birmingham Administrative Court Daily Cause List", true, false),
```

Comment:
Not sure what is the right value for hasExcel and hasAdditionalPdf. Most of the non-strategic lists have them as false and true. You are setting it as true and false for these new lists. This feels a bit wrong because you are generating an Excel file for these lists to include in the subscription emails, while the original input is already an Excel.

Reply:
as discussed, it's updated.